server/datastore/mysql/vpp_test.go

```diff
@@ -355,3 +407,76 @@ func testVPPApps(t *testing.T, ds *Datastore) {
 	require.Equal(t, "foo", appTitles[0].Name)
 	require.Equal(t, app2.Name, appTitles[1].Name)
 }
+
+func testGetVPPAppByTeamAndTitleID(t *testing.T, ds *Datastore) {
+	ctx := context.Background()
+	team, err := ds.NewTeam(ctx, &fleet.Team{Name: "team 2"})
+	require.NoError(t, err)
+
+	// TODO(roberto): replace with actual datastore method(s) once we have them
```

Comment:
Is this going to be part of the cleanup ticket?

Reply:
@gillespi314 yep, added an item.